### PR TITLE
fix: Regression bug with the initial source selection

### PR DESCRIPTION
**Pull request by mcarriere**
The if (item.source.src !== src) had to be removed for the initial selection to be applied

### CODE REVIEW

**Comment by mcarriere** [comment]
My previous code worked fine with the demo.html but once I updated to the latest version on our sites, I noticed a bug that was reintroduced by my latest changes during the code review.

**Comment by jthomerson** [comment]
@yokuze please review.

File: docs/demo/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
    <title>videojs-quality-selector Demo</title>
    <link href="https://unpkg.com/video.js@7.5.4/dist/video-js.css" rel="stylesheet">
    <script src="https://unpkg.com/video.js@7.5.4/dist/video.js"></script>
-   <script src="../../dist/js/silvermine-videojs-quality-selector.min.js"></script>
+   <script src="../../dist/js/silvermine-videojs-quality-selector.js"></script>
```
Comment (yokuze) [inline docs/demo/index.html]: This change does not seem relevant to the fix that this PR is addressing. Although, I agree that it may be nice to use the unminified version in the demo for easier debug-ability, could you remove this change so that this PR only contains changes relevant to the fix?

File: package-lock.json [hunk 2]
```diff
@@ -2613,14 +2613,14 @@
       "dev": true
     },
     "fsevents": {
-      "version": "1.2.4",
-      "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.4.tgz",
-      "integrity": "sha512-z8H8/diyk76B7q5wg+Ud0+CqzcAF3mBBI/bA5ne5zrRUUIvNkJY//D3BqyH571KuAC4Nr7Rw7CjWX4r0y9DvNg==",
+      "version": "1.2.9",
```
Comment (yokuze) [inline package-lock.json]: What are all of these changes to the package-lock.json file for? They also don't seem relevant to the fix. Could we remove them if they are not necessary?

File: docs/demo/index.html (hunk 1 above)
Comment (mcarriere) [reply docs/demo/index.html]: no problem, I removed it

File: package-lock.json (hunk 2 above)
Comment (mcarriere) [reply package-lock.json]: no problem, I removed it

**Comment by jthomerson** [comment]
`@silvermine/videojs-quality-selector@1.2.2` is published containing this change.
